### PR TITLE
Week1 Homework - Oguzhan

### PR DESCRIPTION
**Pull request by oguzhanturgut**
- index.html Basic page layout
- style.css Simple styling for book listing with images
- app.js  createBookList() and createImgSrc() for DOM Manipulation using data objects

### CODE REVIEW

File: Week1/homework/app.js [hunk 1]
```diff
@@ -1,11 +1,144 @@
...
+    crime_and_punishment: './img/crime_and_punishment.jpg',
+    my_name_is_red: './img/my_name_is_red.jpg',
+    to_kill_a_mockingbird: './img/to_kill_a_mockingbird.jpg',
+    the_handmaids_tale: './img/the_handmaids_tale.jpg',
+    chess: './img/chess.png',
+    the_atlas_of_misty_continents: './img/the_atlas_of_misty_continents.jpg',
+    lord_of_the_flies: './img/lord_of_the_flies.jpg',
+  };
+
+  const getBookListContainer = () => document.getElementById('bookList');
```
Comment (nudge) [inline Week1/homework/app.js]: Its not common to use a function when a variable assignment would have worked, but this is fine ;)

File: Week1/homework/app.js [hunk 2]
```diff
@@ -1,11 +1,144 @@
...
+
+  // Creates and appends element(string) to a specific parent(Node)
+  const createAppendElement = (element, parent) => {
+    const el = document.createElement(element);
+    parent.appendChild(el);
+    return el;
+  };
+
+  // Creates Book List UL
+  const createBookList = obj => {
```
Comment (nudge) [inline Week1/homework/app.js]: Nit: Consider naming that is more descriptive than `obj` (same for other places where you use `obj`), perhaps `book` works better in this particular case.

File: Week1/homework/app.js [hunk 3]
```diff
@@ -1,11 +1,144 @@
...
+
+  // Creates img element sourcing from Book images object, returns element
+  const createImgSrc = (obj, id) => {
+    const imgEl = document.createElement('img');
+    imgEl.setAttribute('src', obj[id]);
+    return imgEl;
+  };
+
+  // Creates and appends element(string) to a specific parent(Node)
+  const createAppendElement = (element, parent) => {
```
Comment (nudge) [inline Week1/homework/app.js]: Nit: `createAndAppendElement()` might be a clearer name.

File: Week1/homework/app.js (hunk 2 above)
Comment (oguzhanturgut) [reply Week1/homework/app.js]: Changed all the `obj` parameters to `book`

File: Week1/homework/app.js (hunk 1 above)
Comment (oguzhanturgut) [reply Week1/homework/app.js]: Not the best practice but I've just tried to make more `function` :)

File: Week1/homework/app.js (hunk 3 above)
Comment (oguzhanturgut) [reply Week1/homework/app.js]: Changed `createAppendElement()` to `createAndAppendElement()`

**Comment by oguzhanturgut** [comment]
> Overall a very nice, well written assignment. Good work!

Thank you for making time to review my assignment.

**Pull request closed without merging**